hardware/simulation/cocotb/cocotb.mk

```diff
@@ -0,0 +1,20 @@
+#Makefile
```

Comment:
Why is this not part of the Makefile? They are in the same directory. Whi file will include this makefile segment ?

Reply:
The idea was that cocotb.mk would be called from the Makefile.
The Makefile is called with "make sim" like one would normaly do.
This way there are no target mixing from the original makefiles and the cocotb makefile included.

Reply:
I don't understand well but will merge and then we discuss. The Makefile is isolated in the coco_tb directory . Pasting coco_tb.mk would be equivalent I think.